### PR TITLE
feat: add connect token options on payload

### PR DESCRIPTION
**Pull request by Gabrielpanga**
Add support to send connect token payload to the endpoint as a passthrough.

### CODE REVIEW

File: api/token.ts [hunk 1]
```diff
@@ -3,11 +3,13 @@ import { PluggyClient } from 'pluggy-sdk';
...
 
-export default function (_: NowRequest, res: NowResponse) {
+export default function (req: NowRequest, res: NowResponse) {
   const pluggyClient = new PluggyClient({
     clientId: PLUGGY_CLIENT_ID,
     clientSecret: PLUGGY_CLIENT_SECRET,
   });
 
-  pluggyClient.createConnectToken().then(data => res.json(data));
+  const { itemId, options } = req.body || {}
```
Comment (fmiras) [inline api/token.ts:12]: I think is not necessary the OR operator
```suggestion
  const { itemId, options } = req.body
```

Comment (fmiras) [inline api/token.ts:12]: Shouldn't we validate the `options` object?

**Review by fmiras — CHANGES_REQUESTED** [review]

File: api/token.ts (hunk 1 above)
Comment (Gabrielpanga) [reply api/token.ts:12]: This is to avoid breaking change on GET requests to this endpoint.

Comment (Gabrielpanga) [reply api/token.ts:12]: Its a passthrough, validations will occur on the endpoint.

Comment (tmilar) [reply api/token.ts:12]: My only concern would be what if the options object is empty or undefined, does it work with no errors? If it does, then I'm ok with it

**Review by tmilar — APPROVED** [review]

File: api/token.ts (hunk 1 above)
Comment (Gabrielpanga) [reply api/token.ts:12]: Yes, everything is optional so for now.

**Pull request merged**